styles.css

```diff
@@ -1,32 +1,77 @@
 /* ****** GENERAL STYLES ******** */
+
```

Comment:
Think about an '*' ruleset.  For any styles that you want to apply to all elements (i.e. display: inline-block, width: 100%, padding: none).  If you have this rule at the top, then you can specify when you want these rules broken.  I typically use something like so: 
`* {
  box-sizing: border-box;
  color: var(--secondary-text);
  font-family: var(--body-font);
  font-size: 1.25rem;
  margin: 0;
  padding: 0;
  width: 100%;
}`